website/cue/reference/components/transforms/route.cue

```diff
@@ -42,12 +42,7 @@ components: transforms: route: {
 							condition will be included in this route.
 							"""
 						required: true
-						type: string: {
-							examples: [
-								#".status_code != 200 && !includes(["info", "debug"], .severity)"#,
-							]
-							syntax: "remap_boolean_expression"
-						}
+						type: condition: syntaxes: [{name: "vrl"}]
```

Comment:
I think this is still not right. `datadog_search` should be supported anywhere we have conditions in Vector. I'd suggest we could just leave out passing along which syntaxes are supported and just render them all until we have a component that has a condition field that only supports a subset of them.

Reply:
As per #10257, it looks like it isn't _yet_ the case that `datadog_search` is supported everywhere. But it sounds like @leebenson is going to pull the strings to make that happen. So I'll rework this PR to reflect that and make sure to merge only after those changes have been made.

Reply:
👍 As noted by https://github.com/vectordotdev/vector/issues/10257#issuecomment-985785321 it is the case that it is supported everywhere already.

Reply:
Ah, derp, I misinterpreted that! Okay, that makes all of this easier.